Lib/unittest/test/testmock/testmock.py

```diff
@@ -38,6 +38,16 @@ def cmeth(cls, a, b, c, d=None): pass
     def smeth(a, b, c, d=None): pass
 
 
+class SomethingElse(object):
+    def __init__(self):
+        self._instance = None
+
+    @property
+    def instance(self):
+        if not self._instance:
+            self._instance = 'object'
+
```

Comment:
```suggestion
        return self._instance
```
See below for added assert.

Reply:
Hi, thank you for the review! I don't see an added assert that makes use of this part of the change, am I missing something?

Reply:
The suggestion that I wrote but did not actually add until now.

Reply:
A-ha, thanks! The new assert makes sense though I'm noticing I think this new line should be unindented one level. I don't see a way I can edit a suggestion when I add it to a batch though.

Reply:
Sigh, sorry, my mistake. I don't know what is wrong with my eyes today.